src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -68,8 +66,12 @@ const MetaTrader = (() => {
     };
 
     const addAllAccounts = () => (
-        new Promise((resolve) => {
+        new Promise((resolve, reject) => {
             BinarySocket.wait('mt5_login_list').then((response) => {
+                if (response.error) {
+                    reject(response.error);
```

Comment:
I think we can't proceed if there's an error; user can't really use the dashboard unless we know what their MT5 accounts are. I suggest using the error function of metatrader to display that error message and ask user to try again.

Reply:
Yeah, that is a great idea, will work on it.